Cargo.toml

```diff
@@ -26,6 +26,7 @@ shellexpand = "2.*"
 simplelog = "0.12.*"
 tokio = "1.*"
 toml = "0.4.*"
+dialoguer = { git = "https://github.com/SIMULATAN/dialoguer", features = [] }
```

Comment:
I think crates.io will not let me `publish` a crate with a git dependency... Maybe it will if it's pinned to a commit? I need to check.
We might have to get your changes merged into `dialoguer`, is there a reason you haven't opened a PR?

Reply:
The main reason is that I kinda just haven't gotten around to it, especially since it wasn't fully approved yet.

However, that isn't the only problem:
My changes are vastly different from what the library usually stands for. While the other components aren't exactly configurable, this one very much is and thus seems a bit out of place (and therefore possibly out of scope).
Additionally, the code is kinda messy and the change is pretty large (~500 lines, although the majority of that is because I created a separate `multi_select_plus` component rather than editing the existing one).

Considering that reasoning, I'll try to see if I can get something going that would have a chance at getting merged upstream, both in terms of code quality and "genericness" (as I said a while back, right now, it's very opinionated, having only what's necessary for this feature to work)

Reply:
Sounds good, I'm sure `dialoguer`'s maintainer will have comments to steer you in the right direction :)

Please link the PR here when you open one